app/services/restream.ts

```diff
@@ -100,6 +98,10 @@ export class RestreamService extends StatefulService<IRestreamState> {
     return this.hostsService.streamlabs;
   }
 
+  get url() {
+    return this.host;
+  }
```

Comment:
it doesn't look like you use this function anywhere

Reply:
Removed